<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="10,1">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>780</width>
         <height>418</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,15,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="CustomListWidget" name="frameListWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="autoScrollMargin">
             <number>16</number>
            </property>
            <property name="flow">
             <enum>QListView::LeftToRight</enum>
            </property>
            <property name="isWrapping" stdset="0">
             <bool>false</bool>
            </property>
            <property name="itemAlignment">
             <set>Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>2</width>
     <height>2</height>
    </size>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_File"/>
   <addaction name="actionPrevious_Frame"/>
   <addaction name="actionNext_Frame"/>
   <addaction name="actionSave_Json"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionFit_Window"/>
   <addaction name="separator"/>
   <addaction name="actionSetAsBase"/>
   <addaction name="actionSetAsSystolic"/>
   <addaction name="separator"/>
   <addaction name="actionSystolic_Detection"/>
   <addaction name="actionSegment"/>
   <addaction name="actionSegment_All"/>
   <addaction name="actionClear"/>
   <addaction name="actionRegression"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
   <addaction name="actionOff"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icons/file.png</normaloff>:/res/icons/file.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="toolTip">
    <string>Open Mce File</string>
   </property>
  </action>
  <action name="actionNext_Frame">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icons/next.png</normaloff>:/res/icons/next.png</iconset>
   </property>
   <property name="text">
    <string>Next Frame</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPrevious_Frame">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icons/previous.png</normaloff>:/res/icons/previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous Frame</string>
   </property>
   <property name="toolTip">
    <string>Previous Frame</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icons/help.png</normaloff>:/res/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icons/zoom-in.png</normaloff>:/res/icons/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icons/zoom-out.png</normaloff>:/res/icons/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionOff">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icons/off.png</normaloff>:/res/icons/off.png</iconset>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
   <property name="toolTip">
    <string>Close App</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icons/undo.png</normaloff>:/res/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icons/redo.png</normaloff>:/res/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionFit_Window">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icons/fit-window.png</normaloff>:/res/icons/fit-window.png</iconset>
   </property>
   <property name="text">
    <string>Fit Window</string>
   </property>
   <property name="toolTip">
    <string>Fit Window</string>
   </property>
  </action>
  <action name="actionOpen_File_2">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionHelp_2">
   <property name="text">
    <string>help</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icons/clear.png</normaloff>:/res/icons/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>clear</string>
   </property>
  </action>
  <action name="actionSetAsSystolic">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icons/frame.png</normaloff>:/res/icons/frame.png</iconset>
   </property>
   <property name="text">
    <string>SetAsSystolic</string>
   </property>
   <property name="toolTip">
    <string>Set this Frame as systolic</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionSegment">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icons/icon.jpg</normaloff>:/res/icons/icon.jpg</iconset>
   </property>
   <property name="text">
    <string>Segment</string>
   </property>
   <property name="toolTip">
    <string>Segment Current Frame</string>
   </property>
  </action>
  <action name="actionSegment_All">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icons/segall.jpg</normaloff>:/res/icons/segall.jpg</iconset>
   </property>
   <property name="text">
    <string>Segment All</string>
   </property>
   <property name="toolTip">
    <string>Segment All Selected Frames</string>
   </property>
  </action>
  <action name="actionSetAsBase">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icons/base.png</normaloff>:/res/icons/base.png</iconset>
   </property>
   <property name="text">
    <string>SetAsBase</string>
   </property>
   <property name="toolTip">
    <string>Set Frame as Base</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionSystolic_Detection">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icons/phase.png</normaloff>:/res/icons/phase.png</iconset>
   </property>
   <property name="text">
    <string>Systolic Detection</string>
   </property>
   <property name="toolTip">
    <string>Systolic Detection</string>
   </property>
  </action>
  <action name="actionSave_Json">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icons/save.png</normaloff>:/res/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save Json</string>
   </property>
  </action>
  <action name="actionRegression">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icons/regression.png</normaloff>:/res/icons/regression.png</iconset>
   </property>
   <property name="text">
    <string>Regression</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomListWidget</class>
   <extends>QListWidget</extends>
   <header>widget/customlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
